### PR TITLE
Modify docker network inspect client to check statusCode instead of s…

### PR DESCRIPTION
**Pull request by haoshuwei**
Modify docker client "docker network inspect" to check statusCode instead of strings contains check

Signed-off-by: Shuwei Hao haoshuwei24@gmail.com

### CODE REVIEW

**Comment by vdemeester** [comment]
LGTM 🐹

File: api/client/network.go [hunk 1]
```diff
@@ -224,14 +225,13 @@ func (cli *DockerCli) CmdNetworkInspect(args ...string) error {
...
+		obj, statusCode, err := readBody(cli.call("GET", "/networks/"+name, nil, nil))
 		if err != nil {
-			if strings.Contains(err.Error(), "not found") {
-				fmt.Fprintf(cli.err, "Error: No such network: %s\n", name)
-			} else {
+			if statusCode != http.StatusNotFound {
 				fmt.Fprintf(cli.err, "%s\n", err)
 			}
 			status = 1
+			fmt.Fprintf(cli.err, "Error: No such network: %s\n", name)
```
Comment (thaJeztah) [inline api/client/network.go]: Shouldn't this also be a if/else, same as in https://github.com/docker/docker/pull/18463?

If you change it, please also swap the if/else, and use `statusCode == http.StatusNotFound`, like I commented in https://github.com/docker/docker/pull/18463/files#r46802028

Comment (vdemeester) [inline api/client/network.go]: ah right :stuck_out_tongue_closed_eyes:

**Comment by haoshuwei** [comment]
Have modified to swap print content under if and else

**Comment by tiborvass** [comment]
LGTM

**Comment by thaJeztah** [comment]
Thanks @haoshuwei!

**Pull request merged**